vogon/settings.py

```diff
@@ -245,3 +245,15 @@
 LOGLEVEL = os.environ.get('LOGLEVEL', 'DEBUG')
 
 SESSION_COOKIE_NAME = 'vogon'
+
+
+# Concept types
+PERSONAL = os.environ.get('PERSONAL', '986a7cc9-c0c1-4720-b344-853f08c136ab')
+CORPORATE = os.environ.get('CORPORATE', '3fc436d0-26e7-472c-94de-0b712b66b3f3')
+GEOGRAPHIC = os.environ.get('GEOGRAPHIC', 'dfc95f97-f128-42ae-b54c-ee40333eae8c')
```

Comment:
the environment variables should be called something that makes it clear what they are for. 'personal' could be anything.